src/components/views/donate/CryptoDonation.tsx

```diff
@@ -343,51 +343,49 @@ const CryptoDonation = (props: {
 								);
 								setGivBackEligible(givBackEligible);
 							}}
-							// TODO disabling custom token to discuss later
-							// onInputChange={(i: string) => {
-							// 	// It's a contract
-							// 	if (i?.length === 42) {
-							// 		try {
-							// 			// setSelectLoading(true);
-							// 			getERC20Info({
-							// 				library,
-							// 				tokenAbi,
-							// 				contractAddress: i,
-							// 				networkId: networkId as number,
-							// 			}).then(pastedToken => {
-							// 				if (!pastedToken) return;
-							// 				const found = erc20List?.find(
-							// 					(t: IProjectAcceptedToken) =>
-							// 						t?.symbol ===
-							// 						pastedToken?.symbol,
-							// 				);
-							// 				!found &&
-							// 					erc20List &&
-							// 					setErc20List([
-							// 						...erc20List,
-							// 						pastedToken,
-							// 					]);
-							// 				setCustomInput(
-							// 					pastedToken?.address,
-							// 				);
-							// 				// setSelectLoading(false);
-							// 			});
-							// 		} catch (error) {
-							// 			// setSelectLoading(false);
-							// 			showToastError(error);
-							// 		}
-							// 	} else {
-							// 		setCustomInput(i);
-							// 		erc20OriginalList?.length > 0 &&
-							// 			setErc20List([...erc20OriginalList]);
-							// 	}
-							// }}
+							onInputChange={(i: string) => {
+								if (isGivingBlockProject) return;
```

Comment:
there may be more projects than givingblock that are not accepting custom tokens so we should fetch supporting custom token from backend. You can add below field to `projectBySlug`

`    organization {
      supportCustomTokens
    }
`

Ref: [https://github.com/giveth/impact-graph/issues/441](https://github.com/giveth/impact-graph/issues/441)